project/Dependencies.scala

```diff
@@ -313,6 +314,29 @@ object Dependencies {
     "org.mock-server" % "mockserver-netty" % mockserverNettyV % Test
   ) ++ betterFilesDependencies
 
+  /*
+  The distro artifact contains the actual impl, but transitively includes OSGI bundles that conflict with assembly:
+  - https://github.com/owlcs/owlapi/wiki/Documentation/45d8f63d055f820c6ac2ca6c4679a2a7b705449b#howto
+  - https://github.com/owlcs/owlapi/issues/455
+  - https://github.com/owlcs/owlapi/issues/603
+
+  jcl-over-slf4j.jar is a replacement for commons-logging 1.1.1. Meanwhile our extensive transitive use of apache's
+  httpclient has been including commons-logging 1.2 for a while. Now the owl api dependency jcl-over-slf4j is
+  conflicting during assembly. As there have been no reported errors AFAIK with commons-logging leaving it in for now.
+  However as we use slf4j for cromwell log configuration the correct thing might actually be to exclude commons-logging
+  whenever importing httpclient and include jcl-over-slf4j. That way we can control all of our logging in one place.
+
+  - https://www.slf4j.org/legacy.html#jclOverSLF4J
+   */
+  val owlApiDependencies = List(
+    "net.sourceforge.owlapi" % "owlapi-distribution" % owlApiV
```

Comment:
nice find!

Reply:
h/t: @rushtong and @davidangb. I think OWL/RDF may be in the running vs JSON for the number of JVM library [options](https://xkcd.com/927/).